src/utils/validateSubmissionFormat.ts

```diff
@@ -0,0 +1,9 @@
+export const validateSubmissionFormat = (message: string) => {
+  const hasBlockOfCode = /```[^`]+```/.test(message);
+  const hasGitHubLink = /github\.com\/[a-zA-Z]/.test(message);
+  const hasLineOfCode = /^`([^`]+)`$/gm.test(message);
+
+  const isValidFormat = (hasBlockOfCode || hasGitHubLink) && !hasLineOfCode;

```

Comment:
instead of saving the result in a variable and returning the variable, just return the result.

Instead of this

```
const isValidFormat = hasBlockOfCode || hasGitHubLink;
return isValidFormat;
```

Do this:

```
return hasBlockOfCode || hasGitHubLink;
```